src/main/java/frc/robot/subsystems/Transporter/TransporterCommands.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.Transporter;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import frc.robot.RobotContainer;
+
+public class TransporterCommands {
+    private static Command getSetMotorOutputCommand(double motorOutput) {
+        return new FunctionalCommand(
+                () -> {
+                },
+                () -> RobotContainer.TRANSPORTER.setVoltageOutput(motorOutput),
+                (interrupted) -> RobotContainer.TRANSPORTER.stopMotors(),
+                () -> false,
+                RobotContainer.TRANSPORTER
+        );
+    }
+
+    public static Command getSetTransporterStateCommand(TransporterConstants.TransporterState targetState) {
+        return getSetMotorOutputCommand(targetState.targetMotorOutput);
+    }
```

Comment:
this is public, and uses `getSetMotorOutputCommand`, therefore should be above it.